htdocs/compta/bank/categ.php

```diff
@@ -118,44 +89,28 @@
 print '<td>'.$langs->trans("Ref").'</td><td colspan="2">'.$langs->trans("Label").'</td>';
 print "</tr>\n";
 
-$sql = "SELECT rowid, label";
```

Comment:
The reason is that updating fetchAll is not enough. We need interaction with the loop at different places when building the output.
Prototype done to implement coming new features with fetchAll just fails and shows it need to much work and complete rewrite at a lot of place, where current prototype found into skeleton is working great and ready with minor change. I prefer to progress with small step.
Probably in a next step, but currently it's too early and it will need a complete different writing so this new code will be wrong too, so i prefer to keep cde that is currently correct and similar to other part of code, this will save time when a solution with a fetchAll will be ready.